evap/evaluation/migrations/0103_questionnaire_visibility.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 2.1.8 on 2019-05-28 15:34
+
+from django.db import migrations, models
+
+
+def fill_questionnaire_visibility(apps, _schema_editor):
+    Questionnaire = apps.get_model("evaluation", "Questionnaire")
+    for questionnaire in Questionnaire.objects.all():
+        if questionnaire.obsolete:
+            questionnaire.visibility = 0
+        elif questionnaire.manager_only:
+            questionnaire.visibility = 1
+        else:
+            questionnaire.visibility = 2
+        questionnaire.save()
+
+
+def remove_questionnaire_visibility(apps, _schema_editor):
+    Questionnaire = apps.get_model("evaluation", "Questionnaire")
+    for questionnaire in Questionnaire.objects.all():
+        if questionnaire.visibility == 0:
+            questionnaire.obsolete = True
+            questionnaire.manager_only = False
+        elif questionnaire.visibility == 1:
+            questionnaire.obsolete = False
+            questionnaire.manager_only = True
+        else:
+            questionnaire.obsolete = False
+            questionnaire.manager_only = False
```

Comment:
questionnaire.obsolete = questionnaire.visibility == 0
questionnaire.manager_only = questionnaire.visibility == 1

also, i think you should be able to import and use the constants... or was there some reason why that doesn't work in migrations?

Reply:
when using the constants this error occurs when migrating: `AttributeError: type object 'Questionnaire' has no attribute 'EDITORS'`. seems like one can only access model attributes. we are using hardcoded values in the other migrations as well.
i could again define the constants in the migration but i think that doesn't really help, what do you say?